docs/overview/persistent_storage.md

```diff
@@ -0,0 +1,5 @@
+# Persistent Storage 
+
+You can store the channel state any way you want - whether Google Drive/iCloud, a local disk, any key-value store/database/a remote server, or any combination of them - we provide a clean API that provides objects which can be serialized into simple binary blobs, and stored in any way you wish.
```

Comment:
The double "provide(s)" is a bit difficult to read here. Technically, the "objects" are part of the "API" as they take the form of the parameter LDK passes to a method that the user implements. For example, the user would implement `chainmonitor::Persist` trait (https://docs.rs/lightning/latest/lightning/chain/chainmonitor/trait.Persist.html) and parameterize their `ChainMonitor` with it, which calls it as needed. So maybe this could be reworded in some way.

Reply:
Do you mean parameterize their `ChannelMonitor` here? As this is what get's passed to `persist_new_channel` & `update_persisted_channel`

Reply:
As in one of the _type parameters_ to the `ChainMonitor` struct is `chainmonitor::Persist`.